torch/csrc/lazy/core/ir.cpp

```diff
@@ -41,6 +41,19 @@ hash_t OpKind::hash() const {
   return StringHash(op.toQualString());
 }
 
+hash_t OperandHashes(const OpList& operands, const hash_t& seed, bool bakeInSizes) {
+  hash_t hash = seed;
+  for (auto& operand : operands) {
+    if (!operand) {
+      hash = HashCombine(hash, static_cast<uint64_t>(kNullOpt));
+      continue;
+    }
+    auto operand_hash = bakeInSizes ? operand.hash_with_sizes() : operand.hash_without_sizes();
```

Comment:
@JackCaoG I don't think the 'hash_with_sizes/without_sizes` here is a big deal since they are APIs in Node:: already and that's the bigger issue.  OK with you to land this and then clean up those methods separately?

Reply:
@wonjoolee95 
We actually have own version of `GetOperandHashes`. I think the constructor is a bit messy right now as we explicitly overwrite the constructor to take `xla::shape` and we currently ignore `lazy::shape`. 

```
Node::Node(OpKind op, OpList operands, std::vector<Shape>&& shapes,
           size_t num_outputs, hash_t hash_seed)
```

will not affect us until we adapt codegen and start passing `lazy::shape` around. I am OK with this change now.

Reply:
Thanks for pointing that out. That is correct, this should be a no-op from XLA's side.